meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb

```diff
@@ -6,11 +6,9 @@ LIC_FILES_CHKSUM = "file://AudioManagerPoC/README;md5=be0fe05c89ff7a2632eb080f72
 
 DEPENDS = "qtbase qtdeclarative qtwebkit"
 
-BRANCH = "Intreprid_stable_branch"
-VERSION = "6.1"
-
+SRCREV = "daf851ee7a41d1b0572c0c95e15f61e427ce97f1"
```

Comment:
> Then I would have to rewrite the other part 

Yes, that's true but now this will also force you to get into always thinking about dividing your changes from the very beginning and plan better.

> I get the point of keeping separate commits but this seems more weird than helpful.

You either get it or you don't. :) We need to have same rules for everyone and every commit so in this case it might not be very helpful to divide but once you start bending the rules, where do you draw the line?